### DISCUSSION TITLE
QT version

### DISCUSSION THREAD

**Discussion by nonetrix** [ghostty-org/ghostty#4024] [Feature Requests, Ideas] [answered]
I prefer QT to GTK personally for reasons I won't get into, a QT version would be great maybe packaged as `ghostty-qt` or something. If this is out of scope, hopefully someone can make one if it's super easy to embed like it seems to be

**Comment by mitchellh** [comment, marked as answer]
Not planned, would love to see someone fork and support this themselves.

**Reply by nonetrix** [reply]
Srry accidentally clicked unmark as answer, good to know 👍

I'll keep this open just in case somebody actually does that

**Comment by linux-universe** [comment]
The good thing about libghostty will be that everyone (with knowledge, of course) can build a terminal on top of it with the toolkits they want.

**Reply by nonetrix** [reply]
Iced/libcosmic would be neat too :)

**Comment by pluiedev** [comment]
For anyone who's in the Ghostty Discord and would like to participate in discussions about a possible Qt app runtime, please head to [the `#Qt apprt` thread](https://discord.com/channels/1005603569187160125/1324203480730177546/1324203540880560231).

**Reply by nonetrix** [reply]
Libcosmic thread along with Android via Termux or Linux container in recent Android betas would be neat

**Reply by 00-kat** [reply]
I don't see the point, they'd get auto-archived due to inactivity.  If you *really* want some place to discuss it, anyone is allowed to create a thread in `#development` so you could make it yourself.
